scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -624,7 +625,10 @@ class FormatWriter(formatOps: FormatOps) {
               val widest = alignmentUnits.maxBy(_.width)
               alignmentUnits.foreach { info =>
                 import info._
-                val tokenLengthGap = widest.separatorLength - separatorLength
+                val tokenLengthGap =
+                  if (style.activeForEdition_2020_03)
```

Comment:
oh, is that all the change from previous behaviour? :)

optional OCD version:
```
  if (!initStyle.activeForEdition_2020_03) 0
  else widest.separatorLength - separatorLength
```

Reply:
Yes, only gaps to compensate tokens length difference :)